goalc/compiler/IR.cpp

```diff
@@ -1058,6 +1058,27 @@ void IR_AsmRet::do_codegen(emitter::ObjectGenerator* gen,
   gen->add_instr(IGen::ret(), irec);
 }
 
+///////////////////////
+// AsmFNop
+///////////////////////
+
+IR_AsmFNop::IR_AsmFNop(bool use_coloring) : IR_Asm(use_coloring) {}
```

Comment:
It's not important, but you could leave out the `use_coloring` stuff for this IR.  It's the system used to assign variables to actual hardware registers. The assembly has a few modes where can either use variables with assembly and the compiler takes care of it (`use_coloring = true`), or you can manually specify registers (`use_coloring = false`).  This instruction doesn't touch variables/registers, so it doesn't need this option.